website/src/utils/getContent.ts

```diff
@@ -0,0 +1,9 @@
+// Util function for fetcing the blog description content
+const getContent = (content: string) => {
```

Comment:
```suggestion
const getContentPreview = (content: string) => {
```

I think `getContentPreview` name would be better here since it's used for preview purspose